### PR TITLE
feat: Customizable WellLog Viewer Layout

### PR DESCRIPTION
**Pull request by Vladimir-Kokin**
- Side panels customization mechanism #1702
- Being able to explicitly set numerical zoom as an input field #1239 
- New story to demonstrate custom layout with numerical zoom input field

### CODE REVIEW

File: react/src/lib/components/SubsurfaceViewer/components/Map.tsx [hunk 1]
```diff
@@ -607,18 +607,6 @@ const Map: React.FC<MapProps> = ({
         }
     }, [selection]);
 
-    useEffect(() => {
```
Comment (hkfb) [inline react/src/lib/components/SubsurfaceViewer/components/Map.tsx:610]: @Vladimir-Kokin could you please explain this change?

File: react/src/lib/components/WellLogViewer/MapAndWellLogViewer.stories.jsx [hunk 2]
```diff
@@ -4,6 +4,8 @@ import { MapAndWellLogViewer } from "./MapAndWellLogViewer";
 
 import exampleData from "../../../demo/example-data/deckgl-map.json";
 import { colorTables } from "@emerson-eps/color-tables";
+//import { ColorTable } from "./components/ColorTableTypes";
```
Comment (hkfb) [inline react/src/lib/components/WellLogViewer/MapAndWellLogViewer.stories.jsx:7]: Maybe we could remove this and other disabled code.

**Comment by hkfb** [comment]
@t0oF-azpn is your team reviewing this?

**Comment by Vladimir-Kokin** [comment]
> @t0oF-azpn is your team reviewing this?

Yes,  @mirisb will review

**Review by mirisb — APPROVED** [review]

File: typescript/packages/well-log-viewer/src/SyncLogViewer.stories.tsx [hunk 3]
```diff
@@ -314,24 +319,24 @@ export const Default: StoryObj<typeof Template> = {
...
-                wellpick: require("../../../../example-data/wellpicks.json")[1], // eslint-disable-line
+                wellpick: wellpick[1],
                 name: "HORIZON",
-                colorTables: require("../../../../example-data/wellpick_colors.json"), // eslint-disable-line
+                colorTables: wellpickColorTable, // eslint-disable-line
                 color: "Stratigraphy",
             },
             {
-                wellpick: require("../../../../example-data/wellpicks.json")[0], // eslint-disable-line
+                wellpick: wellpick[0],
```
[the remark this answers is not in the document]
Comment (Vladimir-Kokin) [reply typescript/packages/well-log-viewer/src/SyncLogViewer.stories.tsx:337]: This JSON file has only two wellpicks

File: react/src/lib/components/WellLogViewer/MapAndWellLogViewer.stories.jsx (hunk 2 above)
Comment (Vladimir-Kokin) [reply react/src/lib/components/WellLogViewer/MapAndWellLogViewer.stories.jsx]: removed

**Pull request merged**